src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -594,6 +604,18 @@ public String getInstructorFeedbackRemindParticularStudentsLink(String courseID,
         return link;
     }
     
+    /**
+     * Retrieves the link to submit the request for remind particular student
+     * @param returnUrl the url to return to after submitting the request
+     * @return submit link with return url appended to it
+     */
+    public String getInstructorFeedbackRemindParticularStudentActionLink(String returnUrl) {
```

Comment:
Ohh. I added Action to differentiate it from `getInstructorFeedbackRemindParticularStudentsLink` because `getInstructorFeedbackRemindParticularStudentsLink` returns the link to generate the modal and `getInstructorFeedbackRemindParticularStudentActionLink` returns the link for the form submit for the form in the modal. So I thought it would be better for them to have different name instead of overloading. So should I keep it and just change the Student to Students or should I remove the Action too? @yj-soh